extras/jackson-support/build.gradle

```diff
@@ -5,6 +5,7 @@ dependencies {
     compile "com.fasterxml.jackson.datatype:jackson-datatype-guava"
     compile "com.fasterxml.jackson.datatype:jackson-datatype-jdk7"
     compile "com.fasterxml.jackson.module:jackson-module-afterburner"
+    compile "com.fasterxml.jackson.datatype:jackson-datatype-jdk8"
```

Comment:
also worthwhile to add `com.fasterxml.jackson.datatype:jackson-datatype-jsr310`, since that fixes #319